src/controllers/main/main.ts

```diff
@@ -655,6 +656,46 @@ export class MainController extends EventEmitter {
     )
   }
 
+  async #handleAccountAdderInitLattice(
+    LatticeKeyIterator: any /* TODO: KeyIterator type mismatch */
+  ) {
+    if (this.accountAdder.isInitialized) this.accountAdder.reset()
+
+    try {
+      const latticeCtrl = this.#externalSignerControllers.lattice
+      if (!latticeCtrl) {
+        const message =
+          'Could not initialize connection with your Lattice1 device. Please try again later or contact Ambire support.'
+        throw new EmittableError({ message, level: 'major', error: new Error(message) })
+      }
+
+      await latticeCtrl.unlock(undefined, undefined, true)
```

Comment:
Optional params look a bit ugly like that, would happily refactor sometime in future :v: